README.md

```diff
@@ -219,16 +219,16 @@ kubectl -n ns1 create  serviceaccount aerospike-operator-controller-manager
 
 ```shell
 kubectl get clusterrolebindings.rbac.authorization.k8s.io  | grep aerospike-kubernetes-operator
-aerospike-kubernetes-operator.v2.5.0-74b946466d                 ClusterRole/aerospike-kubernetes-operator.v2.5.0-74b946466d   41m
+aerospike-kubernetes-operator.v3.0.0-74b946466d                 ClusterRole/aerospike-kubernetes-operator.v3.0.0-74b946466d   41m
 ```
 
-In the example above the name of the cluster role binding is `aerospike-kubernetes-operator.v2.5.0-74b946466d`
+In the example above the name of the cluster role binding is `aerospike-kubernetes-operator.v3.0.0-74b946466d`
 
 Edit the role binding and add a new subject entry for the service account
```

Comment:
We need to make `clusterrolebindings ` doc changes here as well, as we will be creating `clusterrolebindings ` instead of editing.

Reply:
Removing this whole section. Create cluster like after this section already cover the namespace RBAC 
creation.